### PR TITLE
Deprecated removal

### PR DESCRIPTION
**Pull request by Kicer86**
Fix for #675

### CODE REVIEW

**Comment by piponazo** [comment]
There is one test failing on the Mac jobs:

```
======================================================================
FAIL: test_run (bugfixes.redmine.test_issue_836.WriteMetadataDestroysResourceForkOnMacOSXForBigFiles)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/build/Exiv2/exiv2/tests/system_tests.py", line 632, in test_run
    self.compare_stderr(i, command, processed_stderr, stderr)
  File "/Users/travis/build/Exiv2/exiv2/tests/system_tests.py", line 755, in compare_stderr
    msg="Standard error does not match"
  File "/Users/travis/build/Exiv2/exiv2/tests/system_tests.py", line 726, in _compare_output
    expected, got, msg=msg
AssertionError: 'Exiv2 exception in modify action for file[177 chars]pe\n' != ''
- Exiv2 exception in modify action for file /Users/travis/build/Exiv2/exiv2/test/data/exiv2-bug836_copy.eps:
- /Users/travis/build/Exiv2/exiv2/test/data/exiv2-bug836_copy.eps: The file contains data of an unknown image type
 : Standard error does not match
```
I guess that test needs to be also removed after removing the EPS image code.

**Comment by Kicer86** [comment]
Yeah, I've been looking at it. I'll try to fix it today.

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,12 +15,10 @@ option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"
 cmake_dependent_option(EXIV2_ENABLE_NLS "Build native language support (requires gettext)" ON "UNIX" OFF)
 option( EXIV2_ENABLE_PRINTUCS2        "Build with Printucs2"                                  ON  )
 option( EXIV2_ENABLE_LENSDATA         "Build including lens data"                             ON  )
-option( EXIV2_ENABLE_VIDEO            "Build video support into library"                      OFF )
```
Comment (piponazo) [inline CMakeLists.txt]: There are two more occurrences of that variable in the code. Could you take care of removing them?

```
cmake/printSummary.cmake
49:OptionOutput( "Building video support:             " EXIV2_ENABLE_VIDEO              )

cmake/packaging.cmake
56:if ( EXIV2_ENABLE_VIDEO )
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -15,12 +15,10 @@ option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"
...
 option( EXIV2_ENABLE_DYNAMIC_RUNTIME  "Use dynamic runtime (used for static libs)"            ON )
 option( EXIV2_ENABLE_WIN_UNICODE      "Use Unicode paths (wstring) on Windows"                OFF )
 option( EXIV2_ENABLE_WEBREADY         "Build webready support into library"                   OFF )
 option( EXIV2_ENABLE_CURL             "USE Libcurl for HttpIo (WEBREADY)"                     OFF )
-option( EXIV2_ENABLE_SSH              "USE Libssh for SshIo (WEBREADY)"                       OFF )
```
Comment (piponazo) [inline CMakeLists.txt]: There are many other occurrences of that variable in the CMake code and documentation. Could you remove the rest of them?

```
14:16 $ ag EXIV2_ENABLE_SSH
README-CONAN.md
608:$ cmake -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_ENABLE_CURL=ON -DEXIV2_ENABLE_SSH=ON ..

cmake/generateConfigFile.cmake
6:    set(EXV_USE_SSH   ${EXIV2_ENABLE_SSH})

cmake/printSummary.cmake
53:    OptionOutput( "USE Libssh for SshIo:               " EXIV2_ENABLE_SSH            )

cmake/findDependencies.cmake
22:    if( EXIV2_ENABLE_SSH )

src/CMakeLists.txt
110:    if( EXIV2_ENABLE_SSH )
170:    if( EXIV2_ENABLE_SSH )

CMakeLists.txt
23:option( EXIV2_ENABLE_SSH              "USE Libssh for SshIo (WEBREADY)"                       OFF )
```

**Review by piponazo — CHANGES_REQUESTED** [review]

**Review by piponazo — APPROVED** [review]

**Comment by Kicer86** [comment]
@piponazo  after @tbeu 's commits it is clean now.

**Pull request merged**